apps/condo/domains/organization/components/CreateOrganizationPageContent.tsx

```diff
@@ -0,0 +1,355 @@
+        ),
+    })
+    const InviteEmployeeDescription = intl.formatMessage({ id: 'organization.createOrganizationForm.invite.description' })
+
+    const client = useApolloClient()
+    const { user, isLoading: userLoading, signOut } = useAuth()
+    const { selectEmployee, employee, isLoading: organizationLoading } = useOrganization()
+    const router = useRouter()
+    const { query: { next }  } = router
+    const isValidNextUrl = next && !Array.isArray(next) && isSafeUrl(next)
+    const redirectUrl = isValidNextUrl ? next : '/'
+
+    const [showOrganizationForm, setShowOrganizationForm] = useState<boolean>(false)
+    const [isCancelModalOpen, setIsCancelModalOpen] = useState<boolean>(false)
+    const [acceptOrRejectInviteLoading, setAcceptOrRejectInviteLoading] = useState<boolean>(false)
+
+    const onError = useMutationErrorHandler()
+    const hasEmployees = useMemo(() => !!employee, [employee])
+    const skipQueryStatement = userLoading || !user || organizationLoading || hasEmployees
+
+    const {
+        data: lastInviteData,
+        loading: lastInviteLoading,
+        refetch: refetchLastInvite,
+    } = useGetLastEmployeeInviteQuery({
+        variables: {
+            userId: user?.id,
+        },
+        onError,
+        skip: skipQueryStatement,
+        // For a better user experience, we display information about invites and organization requests immediately
+        fetchPolicy: 'network-only',
+    })
+    const lastInvite = useMemo(() => lastInviteData?.employees?.filter(Boolean)?.[0], [lastInviteData?.employees])
+
+    const {
+        data: lastOrganizationEmployeeRequestData,
+        loading: lastOrganizationEmployeeRequestLoading,
+        refetch: refetchLastOrganizationEmployeeRequest,
+    } = useGetLastUserOrganizationEmployeeRequestQuery({
+        variables: {
+            userId: user?.id,
+        },
+        onError,
+        skip: skipQueryStatement || lastInviteLoading,
+        fetchPolicy: 'network-only',
+    })
+    const lastOrganizationEmployeeRequest = useMemo(() => lastOrganizationEmployeeRequestData?.requests?.filter(Boolean)?.[0], [lastOrganizationEmployeeRequestData?.requests])
+    const hasRequestsLimitError = useMemo(() => lastOrganizationEmployeeRequest?.retries >= MAX_ORGANIZATION_EMPLOYEE_REQUEST_RETRIES - 1, [lastOrganizationEmployeeRequest?.retries])
+
+    const [acceptOrRejectInvite] = useAcceptOrRejectOrganizationInviteMutation({ onError })
+    const [sendOrganizationEmployeeRequest] = useSendOrganizationEmployeeRequestMutation({ onError })
+
+    const handleAcceptOrReject = useCallback(async (isAccepted: boolean, isRejected: boolean) => {
+        setAcceptOrRejectInviteLoading(true)
+
+        const result = await acceptOrRejectInvite({
+            variables: {
+                id: lastInvite?.id,
+                data: {
+                    dv: 1,
+                    sender: getClientSideSenderInfo(),
+                    ...pickBy({ isAccepted, isRejected }),
```

Comment:
What is the use of `pickBy` here?

Reply:
For filter only exists properties of object